### ISSUE TITLE
Custom elements inside jQuery document fragment are not initialized in IE11

### ISSUE THREAD

**Issue by Dmitri-Sintsov** [ungap/custom-elements#13]
To make custom elements work with jQuery templates (DOM nodes originally detached from `document`), I use the following code:

        $template.appendTo('body');
        $template.detach();

It works fine with browsers that have custom elements v1 internally implemented, custom elements are properly created when appended to `body`. The polyfill, however, does not render custom elements inside these jQuery template fragments.

Is there some workaround (eg. some function call) that could be used to create custom elements inside `template` detached DOM nodes?
